server/controllers/lettersControllers/lettersControllers.js

```diff
@@ -40,8 +40,9 @@ exports.getAllLetters = async (req, res) => {
 
 exports.getLettersById = async (req, res) => {
     try {
-        const {letterId} = req.params.id
+        const letterId = req.params.id
         const letters = await Letters.findById(letterId).exec()
+        res.json(letters)
```

Comment:
We weren't sending a response when looking up letters. Added that functionality.